frontend/src/component/onboarding/ConnectSdkDialog.tsx

```diff
@@ -68,25 +68,48 @@ export const ConnectSdkDialog = ({
 }: IConnectSDKDialogProps) => {
     const theme = useTheme();
     const isLargeScreen = useMediaQuery(theme.breakpoints.up('lg'));
+    const [sdk, setSdk] = useState<Sdk | null>(null);
```

Comment:
parent component will keep track of sdk and env because we need to remember you previous setting when going back and forth